staging/src/k8s.io/api/flowcontrol/v1alpha1/types.go

```diff
@@ -367,14 +375,50 @@ type QueuingConfiguration struct {
 	// +optional
 	AssuredConcurrencyShares int32 `json:"assuredConcurrencyShares" protobuf:"varint,1,opt,name=assuredConcurrencyShares"`
 
+	// `shaper` indicates what to do with requests that can not be executed right now
+	Shaper ShaperConfiguration `json:"shaper,omitempty" protobuf:"bytes,2,opt,name=shaper"`
```

Comment:
`limitResponse` sounds "easier" than `shaper` to me.. shaper reminds of other terms traffic-shaping in networking..

Reply:
Yes, this was intended to evoke the networking metaphor.
If others find "limitResponse" more intuitive then that's OK with me.